### ISSUE TITLE
Kinsta support

### ISSUE THREAD

**Issue by retlehs** [roots/trellis-cli#13]
support making the necessary changes to a trellis install in order to deploy to kinsta

ref [https://kinsta.com/blog/bedrock-trellis/](https://kinsta.com/blog/bedrock-trellis/?kaid=OFDHAJIXUDIV)

**Comment by codepuncher** [comment]
Are there any thoughts on how this should be done @swalkinshaw?

e.g.
1. create a `trellis kinsta patch` subcommand
2. make required changes
3. commit changes with message "patch: Kinsta support"

**Comment by swalkinshaw** [comment]
I don't think this should be built into trellis-cli. A separate extension would make sense though and could do what you proposed which is a good idea 👍 

Maybe it could even use Kinsta's API to fetch some data programmatically? (if they support that)
